Encode.xs

```diff
@@ -139,6 +140,31 @@ do_fallback_cb(pTHX_ UV ch, SV *fallback_cb)
 }
 
 static SV *
+do_bytes_fallback_cb(pTHX_ U8 *s, STRLEN slen, SV *fallback_cb)
+{
+    dSP;
+    int argc;
+    STRLEN i;
+    SV *retval = newSVpvn("",0);
```

Comment:
`retval = newSVsv(POPs);` prior `FREETMPS` would be correct. You could also make it mortal (`sv_2mortal()`) prior returning the retval and remove the call to `SvREFCNT_dec`  in the caller code, this would make the code a bit safer since it's always preferable to work with mortal SV's if possible, on a second look this would require a bit of refactoring.